hibp.js

```diff
@@ -38,6 +46,22 @@ const HIBP = {
     }
     return foundBreaches.filter(({meta}) => meta.IsVerified && !meta.IsRetired && !meta.IsSensitive && !meta.IsSpamList);
   },
+
+  async subscribeHash(sha1) {
+    const sha1Prefix = sha1.slice(0, 6).toUpperCase();
```

Comment:
This is now duplicated in at least 2 places (here and L29 above).
Not sure if we should move the `sha1.slice(0, 6).toUpperCase()` logic somewhere to avoid stuff getting out of sync.